skypy/galaxy/redshift.py

```diff
@@ -123,11 +127,13 @@ def schechter_lf_redshift(redshift, M_star, phi_star, alpha, m_lim, fsky, cosmol
     the sky.
 
     >>> from skypy.galaxy.redshift import schechter_lf_redshift
+    >>> from astropy import units
     >>> z = [0., 5.]
     >>> M_star = -20.5
     >>> phi_star = 3.5e-3
     >>> alpha = -1.3
-    >>> z_gal = schechter_lf_redshift(z, M_star, phi_star, alpha, 22, 1/41253)
+    >>> sky_area = 1 * units.degree * units.degree
```

Comment:
better to construct the deg2 unit than to multiply 1 degree time the degree unit 

```suggestion
    >>> sky_area = 1*units.deg**2
```